devel/qbe/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                qbe
+version             1.0
+categories          devel
+license             MIT
+maintainers         {@alterae} \
+                    openmaintainer
+description         Compiler backend
+long_description    QBE is a compiler backend that aims to provide 70% of the performance of industrial optimizing compilers in 10% of the code.
+homepage            https://c9x.me/compile/
+master_sites        https://c9x.me/compile/release/
+distfiles           qbe-${version}.tar.xz
```

Comment:
you shouldn't need to explicitly specify this, instead add the line `use_xz yes` which specifies the archive type and MacPorts should take care of the rest